src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -244,14 +253,14 @@
            (click)="toggleSubMenu('rgw')"
            [attr.aria-expanded]="displayedSubMenu == 'rgw'"
            aria-controls="collapseBasic">
-          <ng-container i18n>Object Gateway</ng-container>
+          <span i18n>Object Storage</span>
```

Comment:
Currently dashboard has Object Gateway but if it's preferred to change it as **Object storage** 
then I feel it's good change Block as **Block storage** and File system as **File storage**
So that in dashboard it self users easily get to know that ceph supports all these storages
what is your opinion @epuertat ?

Reply:
@sunilangadi2 I get your point and makes a lot of sense. I didn't shorten "Object Storage" to just "Object" because "object" is a too widespread/overused term in engineering, and ["Object storage" seems to be a single term].(https://en.wikipedia.org/wiki/Object_storage). By adding "storage" to everything, wouldn't it be a bit redundant?:
* Block storage
* File storage
* Object storage

I'll think about it.

Reply:
@epuertat Yes even i feel it's bit redundant, I am ok with your views.